shared/src/test/scala/squants/thermal/TemperatureSpec.scala

```diff
@@ -212,21 +245,30 @@ class TemperatureSpec extends FlatSpec with Matchers {
     Kelvin(10).toString should be("10.0°K")
     Fahrenheit(10).toString should be("10.0°F")
     Celsius(10).toString should be("10.0°C")
+    Rankine(10).toString should be("10.0°R")
 
     val k = Kelvin(0)
     k.toString(Kelvin) should be("0.0°K")
     k.toString(Fahrenheit) should be("-459.67°F")
     k.toString(Celsius) should be("-273.15°C")
+    k.toString(Rankine) should be("0.0°R")
 
     val c = Celsius(0)
     c.toString(Kelvin) should be("273.15°K")
     c.toString(Fahrenheit) should be("32.0°F")
     c.toString(Celsius) should be("0.0°C")
+    c.toString(Rankine) should be("491.66999999999996°R")
```

Comment:
No, but the other examples convert cleanly to two decimals. 0C converts to 491.67R, but b/c of floating point arithmetic I had to include the full version. Since it's a string, I couldn't use tolerance.